### DISCUSSION TITLE
A quest to fetch contact information for Places

### DISCUSSION THREAD

**Discussion by ymollard** [streetcomplete/StreetComplete#5863] [Ideas]
Hi,

When solving a "opening_hours" quest in front of a place, it is usually easy to record also the contact information. Would it be possible to add a quest dedicated to query telephone numbers?

e.g. if no `phone=` attribute is found, a "Contact information" quest can ask "Are there means to contact this place? Add a phone number / Add an e-mail"

**Comment by Helium314** [comment]
This was already declined, see #454

**Reply by mnalis** [reply]
As a possible workaround for more advanced users (for who spam is not an issue), they might be interested to know that [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) (StreetComplete _"Expert Edition"_ fork) does have quests asking for `phone` and `website` on various amenities/shops (a list of which one can configure if needed).

**Reply by ymollard** [reply]
Oh I see, thanks for the info.

Actually the team I'm mapping with (and I) are beginners so I prefer not to use the Expert Edition.

For my country too, phone numbers of places are must-haves so it is strange for us to use an "expert edition" of the app to map phone numbers.

Should this no-phone-quest be inserted into the FAQ so that users of such countries understand why there are not quested to fill the phone numbers?

**Reply by HolgerJeromin** [reply]
> For my country too, phone numbers of places are must-haves...

Visible from the street?

**Reply by mnalis** [reply]
> Actually the team I'm mapping with (and I) are beginners so I prefer not to use the Expert Edition.

Understood (although, [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) will look very much like regular StreetComplete until you start enabling more and more functionality in settings. If you only enable two extra quests, I don't think you'd notice much other differences). 

Another way to add phone,email,website etc. data is to use an editor app which allows to add more information if wanted, like [Every Door](https://wiki.openstreetmap.org/wiki/Every_Door) mobile editor -  it has different UI (i.e. point-and-click, instead of quest-based), but is otherwise IMHO not hard to learn. You could also use StreetComplete for most quests, and only use EveryDoor for things you cannot otherwise answer in StreetComplete (but then again, some people dislike using two different apps, so YMMV)

---

However, if you'd prefer to **stick to StreetComplete** (perfectly good choice!), you have the alternative of **leaving Notes**. You can leave them in two ways:

- when you are asked any quest about that place (e.g. "opening hours"), you can answer `Uh...` / `Can't say...` / `Leave note`, and then leave a text note with picture(s) of the door stickers showing all the relevant information (opening hours, phone, website, email, facebook, ....) which will be linked to the place automatically (but beware that once you do that, no further quests will be asked about that amenity!), 
_or_
- you can solve all the quests normally for that amenity, and then long press at its location on the screen and choose `Create new note` and it will create Note here. That will allow you to enter Note without _"sacrificing"_ answering quest(s) in order to do so, but then you have to make sure to identify the place name yourself in the Note text, otherwise it might be confusing which shop/amenity you are talking about if there are several nearby. 

The **disadvantage of leaving Notes is that they do not contribute directly**, i.e. you or other mappers would _later_ have to look at them, decide its worth their time to add that information to the map using some other, full-fledged editor (like iD, JOSM, Vespucci...), and only when they do will the information be available to regular users. 

Depending on activity of Note-solving mappers in your particular area,  if you depend on someone else to do it, that can take from just few hours to more than few years (by which time the information has probably become outdated and irrelevant).  So the only way to be sure that they are added promptly is to do it yourself when you get back home from mapping. 

They say [iD](https://wiki.openstreetmap.org/wiki/ID) is the easiest editor to do that (it runs in a regular web browser when you press `Edit` button on the https://www.openstreetmap.org, so no installation necessary, and includes an easy tutorial on first run).  If you're going to be solving Notes of your fellow team members, tools like [My-notes](https://my-notes.osm-hr.org/) might help to find and solve all of their Notes too.

> Should this no-phone-quest be inserted into the FAQ so that users of such countries understand why there are not quested to fill the phone numbers?

Well, perhaps, but there are _many_ suggested quests there are rejected for inclusion in StreetComplete; it would be impractical to include explanations for each of them in the [FAQ](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ). 

The criteria for inclusion are mentioned in detail in [QUEST_GUIDELINES](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md). Perhaps adding some generic FAQ entry about suggesting new quests (linking to that document) might help? Would you @ymollard have found / read such FAQ entry before opening this discussion? Also, when people choose to open a new issue of type `Quest suggestion` (instead of a new discussion), they are [already](https://github.com/streetcomplete/StreetComplete/issues/new?assignees=&labels=&projects=&template=quest-suggestion.md) linked to those instruction and have to click various checkboxes to confirm applicability.

**Reply by matkoniecz** [reply]
I would not leave notes blindly, without knowing is such info already added.

EveryDoor is probably more fitting if someone wants to provide thorough POI data, including info that is commonly unsigned but sometimes appearing.

**Reply by ymollard** [reply]
Thank you all for your details. In my previous team quests, we have opted for notes too, each time we were quested for opening hours.
@mnalis I hadn't looked into SC git repo tbh, so I haven't found the quest guidelines. But yes, a generic question "Can we add a quest about X" in the FAQ, redirecting to the quest guidelines, could be great to prevent further questions

**Reply by mnalis** [reply]
> I would not leave notes blindly, without knowing is such info already added.

True, but if the SC asks about `opening_hours`, we know at least that should be added/updated, so leaving a Note which mentions _both_ `opening_hours` and other info (`phone`/`website`/`email` etc.) should always be fine, I think.

> But yes, a generic question "Can we add a quest about X" in the FAQ, redirecting to the quest guidelines, could be great to prevent further questions

Thanks for feedback @ymollard, I've now added ["SC FAQ: I have an idea for new Quest, how do I suggest it?"](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#I_have_an_idea_for_new_Quest,_how_do_I_suggest_it?)

**Comment by ymollard** [comment]
Yes @HolgerJeromin, in France, most shops have a phone number and show it clearly on their facade. That is why it is surprising for us not to be quested the phone number by SC when we are quested for their opening schedule.

Here are examples all showing their phone number on their facade, that I shot yesterday in the same street:

Employment agency / Public school / Lawyer + accountant agencies / Insurance agency:
![output_1725269978](https://github.com/user-attachments/assets/30f4119f-d1b9-4f43-ab65-e34a2a7f88e7)

Beauty institute / Canine institute / medical offices / housework agency:
![output_1725269979](https://github.com/user-attachments/assets/1b187ed9-b0d4-44df-b57a-5ac4abf74c6f)

Eye specialist / Hair dresser / Wedding agency / Pizzeria:
![output_1725269980](https://github.com/user-attachments/assets/045d0874-f423-41ed-855d-00a020780f3b)

**Reply by matkoniecz** [reply]
Is it standard across your country that POI has such phone info? 

If yes, how often phone number is missing? In 5% of cases? 10%? 50%?

If yes, which one?

**Reply by ymollard** [reply]
Yes it is standard in the whole country. I would estimate that 30% of POIs don't show phone number in facade. Most bars/pubs don't, since it is not very useful to phone them. Ironically, a lot of restaurants don't show them either, although it'd be very useful to book a table (but some do on their outdoor menu or facade)
